src/adapters/stargate/tests/snapshots/ethereum.profits.json

```diff
@@ -111,19 +111,19 @@
               "name": "Dai Stablecoin",
               "symbol": "DAI",
               "decimals": 18,
-              "profitRaw": "-18179952999981820051n",
+              "profitRaw": "-3229072000000n",
               "type": "underlying",
               "calculationData": {
-                "withdrawalsRaw": "18179949n",
-                "withdrawals": "0.000000000018179949",
+                "withdrawalsRaw": "18179949770928000000n",
+                "withdrawals": "18.179949770928",
```

Comment:
This is the correct value according to what occurred in this tx (https://etherscan.io/tx/0x110172990ef6a644b958a07051d2bc8b52e17db3c39f2a1cdbf7d8686c598e52)

The following command isolates all the withdrawals for this protocol token within the blocks specified.

```
npm run withdrawals 0xB0D502E938ed5f4df2E681fE6E419ff29631d62b 18156819 18163965 0x0faf1d2d3ced330824de3b8200fc8dc6e397850d pool stargate 1 12345
```

The wrong value was due to `SimplePoolAdapter` movements calculation dividing by the wrong number of decimals.

Reply:
thanks for checking